draft-ietf-tls-esni.md

```diff
@@ -100,8 +100,8 @@ anonymity set. Usage of this mechanism reveals that a client is connecting to a
 particular service provider, but does not reveal which server from the
 anonymity set terminates the connection.
 
-ECH is only supported with (D)TLS 1.3 {{!RFC8446}} and newer versions of the
-protocol.
+ECH is only supported with TLS 1.3 {{!RFC8446}}, DTLS 1.3 {{!RFC9147}}, and
+newer versions of the TLS and DTLS protocols.
 
```

Comment:
Maybe:
`ECH is supported in TLS 1.3, DTLS 1.3, and newer versions of the TLS and DTLS protocols.`

Rationale: "only supported with" sounds wrong, and Section 6.1 prohibits TLS 1.2 anyway.